docs/design/index.md

```diff
@@ -40,8 +40,25 @@ The downside of the microautonomy architecture is that the computational perform
 
 ### Autoware concepts
 
+The concept of the Autoware is available [here](autoware-concepts/index.md).
+
+- Target readers:
+- Purpose:
+
+### Autoware architecture
+
+The overall architecture of Autoware and design overview of each component are available [here](autoware-architecture/index.md).
+
+- Target readers: all users and beginner developers
+- Purpose: describe high-level understanding
+
 ### Autoware interfaces
 
+The detailed interface design of each component are available [here](autoware-interfaces/index.md).
+
+- Target readers: intermediate developers
+- Purpose: describe the information of how-to-develop
```

Comment:
By the way, since this is just my comment for confirmation, I think you can delete it.
https://github.com/autowarefoundation/autoware-documentation/pull/86/files#r856877533

Instead, we can write several sentences in other PRs.

```suggestion
### Autoware interfaces

The detailed interface design of each component are available [here](autoware-interfaces/index.md).
```